x-pack/plugins/security_solution/public/onboarding/components/onboarding_body/hooks/use_completed_cards.test.ts

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { renderHook, act, type RenderHookResult } from '@testing-library/react-hooks';
+import { waitFor } from '@testing-library/react';
+import { useCompletedCards } from './use_completed_cards';
+import type { OnboardingGroupConfig } from '../../../types';
+import type { OnboardingCardId } from '../../../constants';
+import { mockReportCardComplete } from '../../__mocks__/onboarding_context_mocks';
+
+const defaultStoredCompletedCardIds: OnboardingCardId[] = [];
+const mockSetStoredCompletedCardIds = jest.fn();
+const mockUseStoredCompletedCardIds = jest.fn(() => [
+  defaultStoredCompletedCardIds,
+  mockSetStoredCompletedCardIds,
+]);
+jest.mock('../../../hooks/use_stored_state', () => ({
+  useStoredCompletedCardIds: () => mockUseStoredCompletedCardIds(),
```

Comment:
`use_stored_state` has multiple exports, jest.requireActual spread is missing to avoid sneaky errors later